tests/gl/canvas2d_test.re

```diff
@@ -0,0 +1,56 @@
+
+/* TODO: make canvas tests here */
+open ReasonJs.Canvas2d;
+open ReasonJs.CanvasElement;
+open ReasonJs.Dom;
+
+/*let canvasEl = DocumentRe.querySelector "canvas" document;*/
+let canvasEl = DocumentRe.createElement "canvas" document;
```

Comment:
Just `Document.createElement` should work here, right? "Usage code" should avoid using internal modules, since they're likely to be hidden in the future.